lib/api/src/modules/stories.ts

```diff
@@ -286,6 +286,11 @@ Did you create a path that uses the separator char accidentally, such as 'Vue <d
       } else if (viewMode && firstLeaf) {
         navigate(`/${viewMode}/${firstLeaf.id}`);
       }
+    } else if (storiesHash[storyId] && storiesHash[storyId].children) {
```

Comment:
@lonyele will this navigate multiple times if `storyId` is not immediately above the leaf?

Reply:
`storiesHash[storyId].children` is not necessarily a leaf, is it? It's just a child node ID. What happens in the case that the child is not a leaf?